unified_planning/engines/factory.py

```diff
@@ -73,6 +73,11 @@
         "unified_planning.engines.compilers.tarski_grounder",
         "TarskiGrounder",
     ),
+    "fast-downward-grounder": ("up_fast_downward", "FastDownwardGrounder"),
+    "fast-downward-reachability-grounder": (
```

Comment:
They should be added also in the `DEFAULT_ENGINES_PREFERENCE_LIST`

Reply:
I added them after the tarski grounder (without any insight which one is better).  I am sure that it is in general a better choice than the up_grounder.